src/components/MapView/Legends/LoadingBar.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { createStyles, WithStyles, withStyles } from '@material-ui/core';
+import { useSelector } from 'react-redux';
+import { LayerKey } from '../../../config/types';
+import { loadingLayerIdsSelector } from '../../../context/mapTileLoadingStateSlice';
+
+function TileLoadingIcon({ classes, layerId }: LoadingBarProps) {
+  const loadingLayerIds = useSelector(loadingLayerIdsSelector);
+  const loading = loadingLayerIds.includes(layerId);
+  return (
+    <div className={classes.container}>
+      <div className={classes.lineBase} />
+      {loading && <div className={classes.line} />}
+    </div>
+  );
+}
+
+const styles = () =>
```

Comment:
Can we simplify this style a bit now that it's stable? No big deal if not but I feel like some of this css is redundant / unused